service/docs/geninfo.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I'm not thrilled that this script depends on files outside of the service directory.  I had worked on trying to replicate the `geopmread --info` and `geopmwrite --info` output with `geopmaccess`, but some information printed by the other command line tools has not been exposed through DBus.  In the future we should try to enable this.  I will open an issue.

Since this is for developers generating documentation rather than for end users, I figure we can let it ride for now.

Reply:
https://github.com/geopm/geopm/issues/1764